ecmascript/parser/tests/test262.rs

```diff
@@ -321,15 +321,13 @@ where
     output
 }
 
-#[test]
```

Comment:
Oops... my local change because I'm too lazy to set this up properly. Will remove.